### ISSUE TITLE
Create a cityGML timelapse for Gratte Ciel

### ISSUE THREAD

**Issue by agrignard** [VCityTeam/UD_ReAgent_ABM#6]
As discussed and agreed with @johnsamuelwrites and @DiegoVinasco we will work on a timelapse for the gratte ciel in the same way as what have been done here. 

https://ud-viz.vcityliris.data.alpha.grandlyon.com/examples/TemporalWidget/example.html

**Comment by johnsamuelwrites** [comment]
The goal will be to create a temporal tileset using the available open data of Gratte Ciel between the years 2009-2018: (precisely, 2009, 2012, 2015, 2018).

**Comment by agrignard** [comment]
Let's first have #7 solved

**Comment by DiegoVinasco** [comment]
I got an initial temporal 3DTiles dataset working with the proposed ROI.

The tileset is online [here](https://dataset-dl.liris.cnrs.fr/three-d-tiles-lyon-metropolis/Villeurbanne_GratteCiel_Temporal_2009-2012-2015-2018_TileSet/) and can be visualized with UD-Viz by pointing the configuration file to the link.

As for the intermediary steps to create the data, they are documented [here](https://github.com/VCityTeam/UD-Reproducibility/tree/grattecieltemporal/Computations/3DTiles/GratteCielTemporal2009-2018) and the data output for each stage of the data pipeline can be found on [nextcloud](https://partage.liris.cnrs.fr/index.php/f/723258567)

That includes the FME workspace I used to extract the original ROI from the metropole datasets.
